implementations/rust/ockam/ockam_command/src/util/api.rs

```diff
@@ -36,11 +36,8 @@ pub(crate) fn query_status() -> Result<Vec<u8>> {
 }
 
 /// Construct a request to query node tcp connections
-pub(crate) fn list_tcp_connections() -> Result<Vec<u8>> {
-    let mut buf = vec![];
-    let builder = Request::get("/node/tcp/connection");
-    builder.encode(&mut buf)?;
-    Ok(buf)
+pub(crate) fn list_tcp_connections() -> RequestBuilder<'static, ()> {
+    Request::get("/node/tcp/connection")
```

Comment:
Nit: it would be great to remove this method an inline the `Request::get("/node/tcp/connection")` at `list.rs` (line 28). These methods are going to be used from one place only, so we are trying to remove them as we do refactors like this.